deploy/cloud/helm/platform/components/operator/templates/secret-env.yaml

```
@@ -21,7 +21,11 @@ metadata:
     {{- include "dynamo-operator.labels" . | nindent 4 }}
 type: Opaque
 stringData:
+  {{ if .Values.dynamo.apiStore.endpoint }}
   API_STORE_ENDPOINT : {{ .Values.dynamo.apiStore.endpoint | quote }}
```

YAMLlint (1.37.1)

[warning] wrong indentation: expected 0 but found 2

(indentation)